File: 2011.执行操作后的变量值/2011-执行操作后的变量值.c
int finalValueAfterOperations(char ** operations, int operationsSize){
    int res = 0;
    for(int i = 0; i < operationsSize; i++)
    {
        if(!(strcmp(*(operations + i), "++X") && strcmp(*(operations + i), "X++")))
        {
            res++;
        }
        else
        {
            res--;
        }
    }

    return res;
}

File: 2011.执行操作后的变量值/2011-执行操作后的变量值.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int res = 0;
        for(int i = 0; i < operations.length; i++)
        {
            switch(operations[i])
            {
                case "++X":
                case "X++":
                    res++;
                    break;

                //只有 4 种操作
                default:
                    res--;
                    break;
            }
        }

        return res;
    }
}